packages/e2e/features/ui/components/authenticator/sign-up-with-attributes.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Sign Up with Attributes
+
+  Amazon Congito User Pools allow for standard OAuth attributes to be required:
+  > https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html
+  These are automatically persisted to `aws-exports.js` and rendered by the Authenticator.
+
+  Background:
+    Given I'm running the example "ui/components/authenticator/sign-up-with-attributes"
+
+  @todo-angular @react @todo-vue
```

Comment:
I think these need to be `@angular @react @vue @todo-angular @todo-vue` for the tests to properly fail on main.

Reply:
```suggestion
  @angular @react @vue @todo-angular @todo-vue
```

Reply:
Oh my gosh you're right. Thanks!